akka-testkit/src/main/scala/akka/testkit/javadsl/TestKit.scala

```diff
@@ -388,9 +388,23 @@ class TestKit(system: ActorSystem) {
    * which uses the configuration entry "akka.test.timefactor".
    */
   @Deprecated
-  @deprecated("Use the overloaded one which accepts java.time.Duration instead.", since = "2.5.12")
+  @deprecated("Use the overloaded one which accepts java.time.Duration instead.", since = "2.5.13")
   def awaitAssert[A](max: Duration, a: Supplier[A]): A = tp.awaitAssert(a.get, max)
 
+  /**
+   * Evaluate the given assert every `interval` until it does not throw an exception and return the
+   * result.
+   *
+   * If the `max` timeout expires the last exception is thrown.
+   *
+   * If no timeout is given, take it from the innermost enclosing `within`
+   * block.
+   *
+   * Note that the timeout is scaled using Duration.dilated,
+   * which uses the configuration entry "akka.test.timefactor".
+   */
+  def awaitAssert[A](max: java.time.Duration, a: Supplier[A]): A = tp.awaitAssert(a.get, max.asScala)
+
```

Comment:
There is one more `awaitAssert` below. Looks like that is missing also.

Reply:
I'm totally fine to add this in the scope of this patch.
Initially I just addressed the issue for the method that was marked deprecated.
I will deprecate the other `awaitAssert` then and add another method signature that will accept `java.time.Duration` parameters as well.

Reply:
thanks